cmd/clair/main.go

```diff
@@ -3,19 +3,16 @@ package main
 import (
 	"context"
 	"flag"
-	"io"
 	golog "log"
-	"net"
-	"net/http"
 	"os"
 	"os/signal"
-	"strings"
 	"time"
 
 	"github.com/rs/zerolog"
 	yaml "gopkg.in/yaml.v2"
 
 	"github.com/quay/clair/v4/config"
+	"github.com/quay/clair/v4/initialize"
```

Comment:
"Initialize" is a weird name for this, as it does *all* the application setup except for calling Listen.

Reply:
Initialization is "get your app ready" main calls listen and serve after init confirms all things are ready to go.